pkgs/development/libraries/x265/default.nix

```diff
@@ -107,7 +111,7 @@ stdenv.mkDerivation rec {
     "-DENABLE_SHARED=${if stdenv.hostPlatform.isStatic then "OFF" else "ON"}"
     "-DHIGH_BIT_DEPTH=OFF"
     "-DENABLE_HDR10_PLUS=ON"
-  ] ++ [
+    (mkFlag (isCross && stdenv.hostPlatform.isAarch) "CROSS_COMPILE_ARM")
```

Comment:
This is odd, with the PR state without this additional x265 fix, x265 cross-compiles fine for armv6, and even cross-compiles fine on current unstable.

 - tested against staging: af1a8d7de6d6cb18ec614330b73150f17c17cf70
 - tested against unstable: 854fdc68881791812eddd33b2fed94b954979a8e

Is this a situation where it fails at runtime?

Reply:
> Is this a situation where it fails at runtime?

Yes, try running the result with `qemu-arm -cpu arm1136`